common/dtd/ldmlSupplemental.dtd

```diff
@@ -457,6 +457,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ATTLIST convertUnit offset CDATA #IMPLIED >
     <!--@MATCH:regex/[-+*/\._ 0-9a-zA-Z]+-->
     <!--@VALUE-->
+<!ATTLIST convertUnit special NMTOKEN #IMPLIED >
+    <!--@MATCH:regex/(100-)?[A-Za-z][-A-Za-z0-9]*-->
```

Comment:
Minor.
since this is a regular ID, it can be a bit simpler.
```suggestion
    <!--@MATCH:regex/[A-Za-z][-A-Za-z0-9]*-->
```

Reply:
Done, thanks